packages/frontend/src/hooks/useETHPriceCharts.tsx

```diff
@@ -168,30 +168,35 @@ export function useETHPriceCharts(initDays = 365, initVolMultiplier = 1.2, initC
     { enabled: ethPrices.isSuccess },
   )
 
-  const squeethPNLSeries = useQuery(
-    ethPriceChartsQueryKeys.squeethPNLSeries(),
-    () => getSqueethPNLCompounding(ethPrices.data ?? [], volMultiplier, collatRatio, days),
-    { enabled: Boolean(ethPrices.isSuccess && days) },
-  )
+  const squeethPNLSeries = useSqueethPNLCompounding(ethPrices.data ?? [], volMultiplier, days)
 
   const longSeries = useMemo(() => {
     return (
-      squeethPNLSeries.data &&
-      squeethPNLSeries.data.map(({ time, longPNL }) => {
+      squeethPNLSeries &&
+      squeethPNLSeries.map(({ time, longPNL }) => {
         return { time, value: longPNL }
       })
     )
-  }, [squeethPNLSeries.data])
+  }, [squeethPNLSeries])
+
+  const squeethIsLive = useMemo(() => {
+    return (
+      squeethPNLSeries &&
+      squeethPNLSeries.map(({ isLive }) => {
```

Comment:
I'm not sure about team's preference but we can use optional chaining here.

Reply:
Could you tell me in details about optional chaining?